### ISSUE TITLE
Certain package versions needed

### ISSUE THREAD

**Issue by nmsutton** [kavli-ntnu/opexebo#44]
I was able to get opexebo to install and report a grid score from a grid cell. I needed to install a python virtual environment and downgrade some packages from the latest versions to get it to work. The latest Python packages versions caused a Numpy error involving use of "np.int" instead of just "int" or something similar to that.

Package versions that work as of 10/07/23 (I don't know which ones are the minimal group necessary so I am listing all in my virtual environment):
```
astropy             4.3
contourpy           1.1.1
cycler              0.12.1
fonttools           4.43.1
imageio             2.31.5
importlib-resources 6.1.0
kiwisolver          1.4.5
lazy-loader         0.3
matplotlib          3.2.0
networkx            3.1
numpy               1.20.0
opencv-python       4.8.1.78
opexebo             0.6.1
packaging           23.2
Pillow              10.0.1
pip                 20.3.4
pkg-resources       0.0.0
pyerfa              2.0.0.3
pyparsing           3.1.1
python-dateutil     2.8.2
PyWavelets          1.4.1
PyYAML              6.0.1
scikit-image        0.18.0
scipy               1.5.0
setuptools          44.1.1
six                 1.16.0
tifffile            2023.9.26
zipp                3.17.0
```

**Comment by simon-ball** [comment]
Fortunate timing! I'm already addressing this in #43, which should be merged Soon (tm)

**Comment by simon-ball** [comment]
If you need it to work Right Now, the problematic element in NumPy - I'm not sure exactly which version finally deprecated the types I'm using, but iirc numpy<=1.22 should behave itself.

**Comment by simon-ball** [comment]
Fixed in v0.7.1, which should be compatible with the most recent versions of NumPy
